modules/installation-create-ingress-dns-records.adoc

```diff
@@ -6,7 +6,7 @@
 [id="installation-create-ingress-dns-records_{context}"]
 = Creating the Ingress DNS Records
 
-If you removed the DNS Zone configuration, manually create DNS records that point to the Ingress load balancer.
+If you removed the DNS Zone configuration, or if you enabled the cluster-wide proxy, manually create DNS records that point to the host name of the router load balancer service.
```

Comment:
I'd say "If you removed the DNS Zone configuration or enabled the cluster-wide proxy," to place equal importance on both options.